### PR TITLE
Work around an issue with github builders

### PR DESCRIPTION
**Pull request by learn-more**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -264,7 +264,7 @@ jobs:
         CFLAGS: /Zi
         CXXFLAGS: /Zi
         LDFLAGS: /DEBUG:FULL /OPT:REF /OPT:ICF
-      run: cmake -Bbuild -Smegasource -T v143 -A ${{ matrix.platform }} --install-prefix %CD%\install -DCMAKE_PDB_OUTPUT_DIRECTORY=%CD%\pdb ${{ steps.vars.outputs.moredef }}
+      run: cmake -Bbuild -Smegasource -T v143 -A ${{ matrix.platform }} --install-prefix %CD%\install -DCMAKE_PDB_OUTPUT_DIRECTORY=%CD%\pdb -DCMAKE_SYSTEM_VERSION=10.0.26100.0 ${{ steps.vars.outputs.moredef }}
```
Comment (MikuAuahDark) [inline .github/workflows/main.yml:267]: Specifying Windows SDK version using `CMAKE_SYSTEM_VERSION` is deprecated. See https://cmake.org/cmake/help/latest/policy/CMP0149.html.

```suggestion
      run: cmake -Bbuild -Smegasource -T v143 -A ${{ matrix.platform }},version=10.0.26100.0 --install-prefix %CD%\install -DCMAKE_PDB_OUTPUT_DIRECTORY=%CD%\pdb ${{ steps.vars.outputs.moredef }}
```

**Review by MikuAuahDark — CHANGES_REQUESTED** [review]

**Comment by slime73** [comment]
Are the arm64 failures in this PR's CI related to the change? I wonder what's needed to fix them

**Review by MikuAuahDark — APPROVED** [review]

**Comment by MikuAuahDark** [comment]
> Are the arm64 failures in this PR's CI related to the change?

No. It's caused by ARM64 misdetection in megasource. Fixed in love2d/megasource@afe097c.

**Comment by slime73** [comment]
Thanks!

**Pull request merged**